packages/babel-types/src/converters/valueToNode.ts

```diff
@@ -104,7 +104,7 @@ function valueToNode(value: unknown): t.Expression {
   // regexes
   if (isRegExp(value)) {
     const pattern = value.source;
-    const flags = /\/([a-z]+|)$/.exec(value.toString())[1];
+    const flags = /\/([a-z]*)$/.exec(value.toString())[1];
```

Comment:
Aside for future PR: regexp.prototype.flags has been supported since node.js 6. We can use `value.flags` here and manually polyfill it with the regex here when compiling for babel-standalone.